internal/services/scrapeService.go

```diff
@@ -0,0 +1,53 @@
+package services
+
+import (
+	"context"
+	"time"
+
+	discoveryv1 "github.com/syncromatics/kafmesh/internal/protos/kafmesh/discovery/v1"
+
+	"github.com/syncromatics/go-kit/log"
```

Comment:
Nice

Reply:
Figured it was a good time to add it in:)